target/linux/ramips/dts/mt7628an_wrtnode_wrtnode2.dtsi

```diff
@@ -71,7 +71,10 @@
 
 &ethernet {
 	mtd-mac-address = <&factory 0x4>;
-	mediatek,portmap = "llllw";
+};
+
+&esw {
+	mediatek,portmap = <0x2f>;
```

Comment:
This seems to be wrong based on 02_network:
```
	wrtnode,wrtnode2p|\
	wrtnode,wrtnode2r|\
	zyxel,keenetic-extra-ii)
		ucidef_add_switch "switch0" \
			"1:lan" "2:lan" "3:lan" "4:lan" "0:wan" "6@eth0"
		;;
```

Reply:
Fixed.